ui/app/AppLayouts/Wallet/views/SavedAddressesView.qml

```diff
@@ -69,6 +70,7 @@ Item {
         StatusListItem {
             id: savedAddress
             title: name
+            objectName: name
```

Comment:
I suggest using sth like `"savedAddress-%1".arg(name)`

Reply:
As this is part of a list view, this is then much easer without any prefix as we can reuse the code more easily to find element into a list